cli/command/load_balancer_test.go

```diff
@@ -0,0 +1,491 @@
+package command
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+func TestAddPortToLoadBalancer(t *testing.T) {
+	base, ctrl := newTestCommand(t)
+	defer ctrl.Finish()
+
+	base.Resolver.EXPECT().
+		Resolve("load_balancer", "lname").
+		Return([]string{"lid"}, nil)
+
+	ports := []models.Port{
+		models.Port{
+			ContainerPort: 80,
+			HostPort:      80,
+			Protocol:      "tcp",
+		},
+	}
+	healthCheck := models.HealthCheck{}
+
+	loadBalancer := &models.LoadBalancer{
+		HealthCheck:      healthCheck,
+		LoadBalancerID:   "lid",
+		LoadBalancerName: "lname",
+		Ports:            ports,
+	}
+
+	base.Client.EXPECT().
+		ReadLoadBalancer("lid").
+		Return(loadBalancer, nil)
+
+	ports = append(ports, models.Port{
+		ContainerPort: 81,
+		HostPort:      81,
+		Protocol:      "tcp",
+	})
+
+	req := models.UpdateLoadBalancerRequest{
+		HealthCheck:    nil,
```

Comment:
[nit] no need to specify a field if it is just going to be its zero value

Reply:
:white_check_mark: